doc/west/index.rst

```diff
@@ -1,14 +1,75 @@
 .. _west:
 
-West (Experimental)
-###################
+West
+####
 
-The Zephyr project is developing a swiss-army knife command line tool
+The Zephyr project maintains a swiss-army knife command line tool
 named ``west``. (Zephyr is an English name for the Greek god of the
 west wind.)
 
-West is developed in its own `repository on GitHub`_. A copy is
-currently maintained in Zephyr's :file:`scripts/meta/west` directory.
+West is developed in its own `repository on GitHub`_.
+
+West is used to obtain the source code for the Zephyr project and can also be
+used to build, debug and flash applications.
+
+.. _west-struct:
+
+Structure
+*********
+
+West is split into two distinct parts that work together to provide the
+full set of functionality offered by the tool:
+
+* Bootstrapper: Installed using ``pip``, implements ``west init``
+* Installation: Installed using ``west init``, implements all other commands
+
+Additional information about the two parts can be found below.
+
+Bootstrapper
+============
+
+The bootstrapper module is distributed using PyPi (i.e. :file:`pip`) and it's
+placed in the user's ``PATH`` becoming the only entry point to west.
+It implements a single command: ``west init``. This command needs to be run
+first in order to use the rest of functionality included in ``west``, since
+``west init`` will do the following:
+
+* Clone West itself in a :file:`west/west` folder
```

Comment:
s/West/west/

Reply:
I'll leave it to you to grep for the rest

Reply:
done